tss-esapi-sys/src/bindings/x86_64-unknown-linux-gnu.rs

```diff
@@ -10798,20 +10774,6 @@ extern "C" {
         nonceTPM: *mut *mut TPM2B_NONCE,
     ) -> TSS2_RC;
 }
-extern "C" {
```

Comment:
There is something strange with these generated files. What version of TSS are these created from and what version do we intend to support in this branch?

Reply:
Just ran the `regenerate-bindings.sh` script which has `TPM2_TSS_VERSION="2.3.3"`. Is there some external dependency that needs to be considered before generating the script?

Reply:
This is unfortunate, it might've been a mistake when we last updated these, but in 2.3.3 these functions are not present. Perhaps we ran with 2.4.0? Given that we'll be doing a minor update on v7, maybe we can bump to the latest v2 of tpm2-tss, 2.4.6. What do you think?